tests/unittests/CoreText/CTTypesetterTests.mm

```diff
@@ -45,15 +45,20 @@
     EXPECT_EQ_MSG(13,
                   index,
                   "SuggestLineBreak should end line at space when soft line break is possible and the text is longer than the width");
+
+    CFRelease(ts);
 }
 
+/* TODO:: 998
 TEST(CTTypeSetter, CreateLineShouldTrimWhitespace) {
```

Comment:
DISABLED_TEST works just as well - it's not as if these tests are failing to build, after all.